src/UISettings.hpp

```diff
@@ -78,6 +78,12 @@ struct UISettings {
   constexpr unsigned GetPercentScale() const noexcept {
     return scale;
   }
+
+  [[gnu::pure]]
+  static bool GetDarkMode(const UISettings &settings) noexcept;
+
+  static uint8_t GetBgColor(const UISettings &settings) noexcept;
```

Comment:
What's the point of declaring these `static` and explicitly passing a `this` pointer under a different name?

Reply:
`GetBgColor()` sound like it returns a `Color` but it returns an integer. This is confusing.

Reply:
So I change declarations to below. Looks right?

`bool GetDarkMode() const noexcept;`

`uint8_t GetBgColorVal() const noexcept;`

When I update my PR, should I always do "git push --force" or just "git push" on my side? Sorry, I am not professional in neither git nor software so let me know when I am out sailing :-).